tmail-backend/tmail-third-party/openpaas/pom.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.linagora.tmail</groupId>
+        <artifactId>tmail-third-party</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>tmail-openpaas</artifactId>
+    <name>Twake Mail :: Third Party :: OpenPaaS</name>
+    <description>OpenPaaS integration for Twake Mail</description>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.mock-server</groupId>
```

Comment:
Please respect alphabetic order for dependencies

Reply:
> Please respect alphabetic order for dependencies

THis comment isn't handled...

Reply:
Yes, I wasn't sure how to handle it, like sort by artifact name or group. I was going to ask about it in our daily meeting.